js/bitopro.js

```diff
@@ -0,0 +1,1151 @@
+        //
+        const result = {
+            'info': response,
+        };
+        for (let i = 0; i < response.length; i++) {
+            const balance = response[i];
+            let currencyId = this.safeString (balance, 'currency');
+            currencyId = currencyId.toUpperCase ();
+            const amount = this.safeString (balance, 'amount');
+            const available = this.safeString (balance, 'available');
+            const account = {
+                'free': available,
+                'total': amount,
+            };
+            result[currencyId] = account;
+        }
+        return this.safeBalance (result);
+    }
+
+    async fetchBalance (params = {}) {
+        await this.loadMarkets ();
+        const response = await this.privateGetAccountsBalance (params);
+        const balances = this.safeValue (response, 'data', []);
+        //
+        //     {
+        //         "data":[
+        //             {
+        //                 "currency":"twd",
+        //                 "amount":"0",
+        //                 "available":"0",
+        //                 "stake":"0",
+        //                 "tradable":true
+        //             }
+        //         ]
+        //     }
+        //
+        return this.parseBalance (balances);
+    }
+
+    parseOrderStatus (status) {
+        const statuses = {
+            '-1': 'open',
+            '0': 'open',
+            '1': 'open',
+            '2': 'closed',
+            '3': 'closed',
+            '4': 'canceled',
+        };
+        return this.safeString (statuses, status, undefined);
+    }
+
+    parseOrder (order, market = undefined) {
+        //
+        // createOrder
+        //         {
+        //             orderId: '2220595581',
+        //             timestamp: '1644896744886',
+        //             action: 'SELL',
+        //             amount: '0.01',
+        //             price: '15000',
+        //             timeInForce: 'GTC'
+        //         }
+        //
+        // fetchOrder
+        //         {
+        //             "id":"8777138788",
+        //             "pair":"bnb_twd",
+        //             "price":"16000",
+        //             "avgExecutionPrice":"0",
+        //             "action":"SELL",
+        //             "type":"LIMIT",
+        //             "timestamp":1644899002598,
+        //             "status":4,
+        //             "originalAmount":"0.01",
+        //             "remainingAmount":"0.01",
+        //             "executedAmount":"0",
+        //             "fee":"0",
+        //             "feeSymbol":"twd",
+        //             "bitoFee":"0",
+        //             "total":"0",
+        //             "seq":"BNBTWD548774666",
+        //             "timeInForce":"GTC",
+        //             "createdTimestamp":1644898944074,
+        //             "updatedTimestamp":1644899002598
+        //         }
+        //
+        let id = this.safeString (order, 'orderId');
```

Comment:
You can do one line here: 

`const id = this.safeString2 (order, 'id', 'orderId');`